CONTRIBUTING.md

```diff
@@ -25,11 +35,78 @@ again.
    use your corporate email address here, not your personal address.
 2. Fork the repository into your own Github account.
 3. Please include unit tests (and integration tests if applicable) for all new code.
-4. Make sure all existing tests pass.
+4. Make sure all existing tests pass (but see the note below about integration tests).
    * In `jib-core`, run `./gradlew clean goJF build integrationTest`
-   * In `jib-gradle-plugin`, run `./gradlew clean goJF build` *
-   * In `jib-maven-plugin`, run `./mvnw clean fmt:format verify` *
+   * In `jib-gradle-plugin`, run `./gradlew clean goJF build integrationTest`
+   * In `jib-maven-plugin`, run `./mvnw clean fmt:format verify -Pintegration-tests`
 5. Associate the change with an existing issue or file a [new issue](../../issues).
 6. Create a pull request!
 
-\* *Note that you will not be able to run the integration tests for `jib-gradle-plugin` or `jib-maven-plugin` because those push to our integration-testing GCP project. If you would like to run integration tests, change all uses of the `jib-integration-testing` project to your own GCP project and run `./gradlew integrationTest` for `jib-gradle-plugin` and `./mvnw verify -Pintegration-tests` for `jib-maven-plugin`.*
+**Note** that you will not be able to run the integration tests for `jib-gradle-plugin` or `jib-maven-plugin` because those push to our integration-testing GCP project. If you would like to run integration tests, change all uses of the `jib-integration-testing` project to your own GCP project and run `./gradlew integrationTest` for `jib-gradle-plugin` and `./mvnw verify -Pintegration-tests` for `jib-maven-plugin`.
+
+# Development Tips
+
+## Debugging the Jib Maven Plugin (`jib-maven-plugin`)
+
+### Build and use a local snapshot 
+
+To use a local build of the `jib-maven-plugin`:
+
+  1. Build and install `jib-core` into your local `~/.m2/repository`
```

Comment:
`jib-core` can't be installed on its own yet (I think you made the correction in the Gradle section but forgot to change it here as well).